calendar-bundle/composer.json

```diff
@@ -34,13 +34,13 @@
         "contao/manager-plugin": "^2.3.1",
         "contao/test-case": "^4.4",
         "phpunit/phpunit": "^8.5",
-        "symfony/http-client": "4.4.* || 5.2.*",
+        "symfony/http-client": "^4.4 || ^5.2",
         "symfony/phpunit-bridge": "^5.2"
     },
     "extra": {
         "contao-manager-plugin": "Contao\\CalendarBundle\\ContaoManager\\Plugin",
         "symfony": {
-            "require": "4.4.* || ^5.2"
+            "require": "^4.4 || ^5.2"
```

Comment:
I do not like these changes, either. We agreed to pin the Symfony minor versions (e.g. `4.4.* || 5.3.*`), because in the past, there were too many (partly unintended) breaking changes. Using `^5.2` not only implies that we have to continue to support Symfony 5.2, it also means that our users will get Symfony 5.4 as soon as it has been released.

Reply:
> it also means that our users will get Symfony 5.4 as soon as it has been released.

I think we made the decision to use `X.X.*` before we had the `contao/conflicts` package. So this should not be an issue anymore. We can very quickly add a conflict for the component that causes issues and have enough time to fix it.

Reply:
> it also means that our users will get Symfony 5.4 as soon as it has been released.

What's wrong with this? In the case, 5.4 does not work as expected, I can easily adjust the constraint.

Reply:
I am fine with changing our policy here. @contao/developers Any objections?

Reply:
As described above, I'm unsure about this. If Contao 4.10 hat `^5.1` set, it would get Sf 5.4 as well and it wouldn't surprise me if that wouldn't work. Yes, you're using an outdated version, but that doesn't mean it should break on an update. Yes, we _could_ manage that through conflicts, but we don't really want to manage conflicts for any previous Contao version? Otherwise we'll need to add conflicts for Contao 4.10 in 4 years when Symfony 5.4 adds an unintended break?

Reply:
> Otherwise we'll need to add conflicts for Contao 4.10 in 4 years when Symfony 5.4 adds an unintended break?

I don't think we ever added conflicts for EOL versions, did we?

Reply:
I personally would never update EOL packages with the expectation that everything will be fine 🙃

Reply:
Yeah, Contao 4.10 will be EOL at that point and the only way to fix it (as with any problem, no matter if it's direct or indirect) is to update to a currently supported version. There are currently also numerous other problems in 4.5, 4.6, 4.7, 4.8 for example that occur through varies updated dependencies. (These issues come up in the community because sometimes people try to update their Contao 4.4 instance to 4.5, 4.6, 4.7, 4.8 first instead of going directly to 4.9 because they think that's what you are supposed to do.)